misc/images/frontegg/mzbuild.yml

```
@@ -0,0 +1,15 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+name: frontegg
```

I think calling this `frontegg-mock` would be less confusing. Also matches the binary name that way!

```suggestion
name: frontegg-mock
```